sql-plugin/src/main/scala/com/nvidia/spark/rapids/literals.scala

```diff
@@ -473,7 +473,7 @@ object GpuScalar extends Logging {
  *
  * This class is introduced because many expressions require both the cudf Scalar and its
  * corresponding Scala value to complete their computations. e.g. 'GpuStringSplit',
- * 'GpuStringLocate', 'GpuDivide', 'GpuDateAddInterval', 'GpuTimeMath' ...
+ * 'GpuStringLocate', 'GpuDivide', 'GpuDateAddInterval', 'GpuTimeAdd' ...
```

Comment:
`GpuTimeMath` was an abstract class being implemented by `GpuTimeAdd` and `GpuDateAddInterval`. I removed it because the reusability of the two classes is actually poor.